testsuite/integration-arquillian/tests/other/webauthn/src/test/java/org/keycloak/testsuite/webauthn/admin/WebAuthnPolicySettingsTest.java

```diff
@@ -275,7 +274,8 @@ public void signatureAlgorithms() {
     public void attestationConveyancePreference() {
         // default not specified
         AttestationConveyancePreference attestation = getPolicyPage().getAttestationConveyancePreference();
-        assertThat(attestation, nullValue());
+        assertThat(attestation, notNullValue());
```

Comment:
This is not expected change. Allow me to check why it happens.

Reply:
I just thought if a user doesn't specify `attestationConveyancePreference`, its representation should be `null`, not `new AttestationConveyancePreference("")`.


But now I understand `WebAuthnPolicy` represent the state "not specified" as empty string. Umm.

Why don't you return `String` instead of `AttestationConveyancePreference` from `WebAuthnPolicyPage#getAttestationConveyancePreference` method? `WebAuthnPolicy#getAttestationConveyancePreference` returns `String`.

Reply:
Yes, I'm about to do that in a follow-up task. As it's not enum anymore :)